### DISCUSSION TITLE
Disconnected image before loading map?

### DISCUSSION THREAD

**Discussion by joq3** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#763] [General]
When I load up my dashboard with the vacuum map I get a image/text showing DISCONNECTED for a breif moment before the map loads up. Is this a default image in this map card or is it my Dreme integration that creates this?

Is it possible to hide this? I rather have no map until it loads than a blurry text reading DISCONNECTED.

Looks like this:
![bild](https://github.com/user-attachments/assets/baa47d21-1b6b-45dd-911e-28b58e47bf43)

**Comment by PiotrMachowski** [comment]
This is a default image that is replaced as soon as real map image is loaded

**Reply by joq3** [reply]
Is it possible to replace the image with a transparent image or hide it completely? I rather have nothing than this image. It is visible for a few seconds sometimes.

**Reply by PiotrMachowski** [reply]
It is hardcoded in the code, so it is not possible for you to change it.

**Reply by joq3** [reply]
Could this be an option in the future. Hiding it with CSS does not help either, as CSS is not always that quick to load.
